composer.json

```diff
@@ -11,12 +11,17 @@
         }
     ],
     "repositories": [
-        { "type": "composer", "url": "https://updates.ez.no/ttl" }
+        { "type": "composer", "url": "https://updates.ez.no/ttl" },
+        { "type": "vcs", "url": "https://github.com/adriendupuis/ezplatform-admin.git" }
```

Comment:
I add to create a [GitHub token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) for Composer with only `read:packages` policy and add it to my [auth.json](https://getcomposer.org/doc/articles/handling-private-packages-with-satis.md#authentication).